src/drc/api/serializers.py

```diff
@@ -138,15 +171,11 @@ class EnkelvoudigInformatieObjectSerializer(serializers.HyperlinkedModelSerializ
         view_name='enkelvoudiginformatieobject-detail',
         lookup_field='uuid'
     )
-    inhoud = AnyBase64File(
-        view_name='enkelvoudiginformatieobject-download',
+    inhoud = ViewFileFile(
+        view_name='enkelvoudiginformatieobject-download', read_only=True,
         help_text=_(f"Minimal accepted size of uploaded file = {settings.MIN_UPLOAD_SIZE} bytes "
```

Comment:
This help text should now be moved to `BestandsDeel.inhoud`

Reply:
But the file size in `BestandsDeel.inhoud` is show in the `grootte` for each object